guides/doc-Upgrading_and_Updating/topics/preparing_to_migrate_pulp_content.adoc

```diff
@@ -15,6 +15,30 @@ During migration to Pulp 3, the amount of data stored in the PostgreSQL data dir
 Ensure that you have enough space for the migration.
 A rough estimation of the space you require is 2-3 times the size of the MongoDB Pulp 2 database.
 
+.Pre-Migration Recommendations
+. Ensure that all enabled repositories are synced properly. If any repositories are synced with a *Warning* or *Error* state, fix the underlying problem and resync the repository.
+
+. Ensure no paused tasks in the system are related to any repositories or content views. If there are any:
+
+* Investigate the underlying problem and resume the task to have it completed successfully.
+
+. If a paused content view task is being cleaned up or has been deleted without fixing the underlying cause, ensure that the related broken version of the content view is being deleted as it is already an unusable version.
+
+. If some content view or Composite content view versions are not required anymore, please consider performing a cleanup of them. This solution article https://access.redhat.com/solutions/2760531 can be referred to understand how to perform the cleanup of older content view/Composite content view versions using `hammer_cli`.
```

Comment:
Content View is always capitalized. 
```suggestion
If you no longer require specific Content Views or Composite Content View Versions, you can perform a cleanup action. 
For more information about how to perform the cleanup, see Red Hat Knowledgebase article https://access.redhat.com/solutions/2760531 .
```
Please add the title of the article and create a proper ascii hyperlink. 
This is not a step. Don't use procedural markup for a statement. 
We avoid the passive tense as much as possible. Maintain an active voice unless it is super awkward to do so. 
We never say please. Refer to IBM style on this. 
We maintain one sentence per line rule. This is in our readme file. 
We don't use slashes in our docs. Read IBM style for guidance on this too. 
Please get used to IBM style conventions.